src/Helpers/helpers.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+if (!function_exists('getEnumValues')) {
```

Comment:
convert this to a [enum method](https://www.php.net/manual/en/language.enumerations.methods.php).